### DISCUSSION TITLE
How chek the query parameters

### DISCUSSION THREAD

**Discussion by julien-h-webcenter** [mockoon/mockoon#734] [Help] [answered]
Hi everybody,

i would like to know if it's possible to check the query parameters and choose a body following the check.
For example i have this url
/home?param=1&param2=2
/home?param=1&param2=3
i would like to answer a different body following the param2 
![Capture d’écran, le 2022-06-17 à 14 16 21](https://user-images.githubusercontent.com/17987431/174355086-19c2703a-5d6b-4e83-bb05-8c806f9246a8.png)
![Capture d’écran, le 2022-06-17 à 14 17 03](https://user-images.githubusercontent.com/17987431/174355162-fb482148-2d37-4ae3-bb67-4300f2690c3b.png)
i see that with have template to use the query params but don't know if we can use inside the rules.

thanks,

Julien

**Comment by 255kb** [comment, marked as answer]
I would just change the rules for response 2 to:
 
![image](https://user-images.githubusercontent.com/7489814/174790561-02750df5-d8a3-4fd2-973f-733c0f436a16.png)

Then, you can provide a different body for each response:
![image](https://user-images.githubusercontent.com/7489814/174790643-bac8fab7-64bd-4247-aad8-5c98a045c4c5.png)

And when you will call the route with either `/home?param=1&param2=2` or `/home?param=1&param2=3` you will get one response or the other.

**Reply by julien-h-webcenter** [reply]
thanks guys 
beginner mistake :). it works now
